tests/free_threading/test_stress.py

```diff
@@ -0,0 +1,180 @@
+import random
+import re
+import threading
+
+import yaml
+
+
+class Dice(tuple):
+    def __new__(cls, a, b):
+        return tuple.__new__(cls, (a, b))
+
+    def __repr__(self):
+        return "Dice(%s,%s)" % self
```

Comment:
This is shared with `test_resolver` - maybe refactor common stuff into e.g. an `utils.py`

Reply:
Done.